hey.go

```diff
@@ -69,9 +69,9 @@ var usage = `Usage: hey [options...] <url>
 
 Options:
   -n  Number of requests to run. Default is 200.
-  -c  Number of requests to run concurrently. Total number of requests cannot
+  -c  Number of workers to run concurrently. Total number of requests cannot
       be smaller than the concurrency level. Default is 50.
-  -q  Rate limit, in queries per second (QPS). Default is no rate limit.
+  -q  Rate limit, in queries per worker per second. Default is no rate limit.
```

Comment:
This is actually per request per worker per second. Is it possible to say "QPS per worker"?

Reply:
Fixed it